_posts/2016-11-13-recurse-center-week-7.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: "Week[6] @ The Recurse Center 🏁"
+comments: true
+categories: tech recurse
+---
+
+A week with the Winter 1s complete! Time for another review.
+
+### Overview
+
+I've started feeling much more focused at RC. The change in people has really helped this. Exited for what the next 5 weeks brings!
```

Comment:
(I keep spelling "exercise" with a "C" - "Excercise". And then I think "NO! NO C AFTER X"... and then I spell things like "exited"...)